pkg/linters/license/license.go

```diff
@@ -40,10 +39,12 @@ func (o *Copyright) Run(m *module.Module) (errors.LintRuleErrorsList, error) {
 
 	result.Merge(OssModuleRule(m.GetName(), m.GetPath()))
 
+	moduleExcludes := getExcludes(o.cfg.CopyrightExcludes, m.GetName())
+
 	for _, fileName := range files {
 		name, _ := strings.CutPrefix(fileName, m.GetPath())
-		name = m.GetName() + ":" + name
```

Comment:
Here we create a name including the module name so that it can be compared with the list of exceptions. The proposed change below does not do this. As a result, we will compare files with exceptions from other modules.